aiven/internal/service/wait.go

```diff
@@ -160,35 +160,6 @@ func WaitAfterStaticIpsDissassociation(ctx context.Context, d *schema.ResourceDa
 	return nil
 }
 
-func WaitBeforeStaticIpsDissassociation(ctx context.Context, d *schema.ResourceData, m interface{}) error {
-	timeout := d.Timeout(schema.TimeoutDelete)
-	log.Printf("[DEBUG] Static Ip dissassociation timeout %.0f minutes", timeout.Minutes())
-
-	conf := &resource.StateChangeConf{
-		Pending:                   []string{"doing"},
-		Target:                    []string{"done"},
-		Delay:                     10 * time.Second,
-		Timeout:                   timeout,
-		MinTimeout:                2 * time.Second,
-		ContinuousTargetOccurence: 5,
-		Refresh: func() (interface{}, string, error) {
-			if rdy, err := staticIpsReadyToBeDissassociated(d, m); err != nil {
-				return nil, "", fmt.Errorf("unable to check if static ips are disassociated: %w", err)
-			} else if !rdy {
-				log.Printf("[DEBUG] still waiting for static ips to be able to be disassociated")
-				return struct{}{}, "doing", nil
-			}
-			return struct{}{}, "done", nil
-		},
-	}
-
-	_, err := conf.WaitForStateContext(ctx)
-	if err != nil {
-		return fmt.Errorf("unable to wait for for static ips to be ready to be dissassociated: %w", err)
```

Comment:
would be nice if we can simplify this `to be ready to be dissassociated` because it is hard to understand what is happening. Also, it looks like there is a typo in `dissassociated`